ktor-features/ktor-auth/jvm/src/io/ktor/auth/ForbiddenResponse.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2014-2019 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.auth
+
+import io.ktor.http.content.*
+import io.ktor.http.*
+import io.ktor.http.auth.*
+
+/**
+ * Response content with `403 Forbidden` status code and `WWW-Authenticate` header of supplied [challenges]
+ * @param challenges to be passed with `WWW-Authenticate` header
+ */
+class ForbiddenResponse(vararg val challenges: HttpAuthHeader) : OutgoingContent.NoContent() {
```

Comment:
Code duplication with `UnauthorizedResponse`. Possible option could be creating 'ChallengeResponse` with status code first parameter, deprecate or inherit `UnauthorizedResponse`.